tests/TestCase/Controller/ControllerTest.php

```diff
@@ -626,7 +626,7 @@ public function testRefererSlash()
 
         $controller = new Controller($request);
         $result = $controller->referer('/some/path', true);
-        $this->assertEquals('/base/some/path', $result);
+        $this->assertEquals('/some/path', $result);
```

Comment:
People never should use that kind of setup IMO ;) But we all know they do.
Is there a way to maybe provide this BC - so you opt in for the "new correct" way?
Maybe with some Configure key then? Or we can apply this for the next 3.3 version with a migration note.